### PR TITLE
[18.05] Fix tool error report history view link

### PR DESCRIPTION
**Pull request by dannon**
xref #6049

### CODE REVIEW

**Comment by dannon** [comment]
It looks like display_common uses the same URL, going to look into where that's used and fix it too.

File: templates/display_common.mako [hunk 1]
```diff
@@ -149,7 +149,7 @@
     %if history.slug and history.user.username:
         <% return h.url_for( controller='/history', action='display_by_username_and_slug', username=history.user.username, slug=history.slug, qualified=qualify ) %>
     %else:
-        <% return h.url_for( controller='/history', action='view', id=trans.security.encode_id( history.id ), qualified=qualify, use_panels=context.get('use_panels', True) ) %>
+        <$ return h.url_for("/histories/view", id=trans.security.encode_id( history.id ), qualified=qualify) %>
```
Comment (martenson) [inline templates/display_common.mako]: I did not see this `$` yet and quick glance over http://docs.makotemplates.org/en/latest/syntax.html does not seem to have it.

Comment (dannon) [reply templates/display_common.mako]: Good catch.  It's not documented because it's a typo.  :(

Comment (martenson) [reply templates/display_common.mako]: It looked liked it *could* be some sort of escaping. 🙃

**Pull request merged**